lua/astrocommunity/color/twilight-nvim/twilight-nvim.lua

```diff
@@ -0,0 +1,4 @@
+return {
+  { "folke/twilight.nvim", cmd = "Twilight" },
+  { import = "astrocommunity.editing-support.zen-mode-nvim" },
+}
```

Comment:
```suggestion
return { "folke/twilight.nvim", cmd = "Twilight" }
```


Does twilight have a strict requirement for `zen-mode`? seems like it just pairs nicely together and people who use twilight don't necessarily want zen-mode

Reply:
Nope it doesn't, thought it was a nice addition but that's fine too.